backtrace-library/src/main/java/backtraceio/library/metrics/EventsPayload.java

```diff
@@ -0,0 +1,45 @@
+package backtraceio.library.metrics;
+
+import android.content.pm.PackageManager;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+import backtraceio.library.logger.BacktraceLogger;
+import backtraceio.library.models.json.BacktraceAttributes;
+
+public abstract class EventsPayload {
+
+    private static transient String LOG_TAG = EventsPayload.class.getSimpleName();
+
+    @SerializedName("application")
+    private final String application;
+
+    @SerializedName("appversion")
+    private final String appVersion;
+
+    @SerializedName("metadata")
+    private EventsMetadata eventsMetadata;
+
+    public EventsPayload(BacktraceAttributes backtraceAttributes, int droppedEvents) {
+        String appVersion;
```

Comment:
Maybe `String appVersion = ""; ` and remove assign in `catch`

Reply:
I think that would be better, thank you!